proto/job.proto

```diff
@@ -0,0 +1,101 @@
+// The file defines the CommandService API.
+syntax = "proto3";
+
+package job;
+
+option go_package = "/job";
+
+import "google/protobuf/timestamp.proto";
+
+// JobService provides methods for starting a job and streaming its output.
+service JobService {
+  // Starts the job and returns an identifier for the job.
+  rpc StartJob(StartRequest) returns (StartResponse) {}
+  // Returns a list of running jobs with their jobId and jobStart timestamps.
+  rpc ListJobs(ListJobsRequest) returns (ListJobsResponse) {}
+  // Stream the output of the job.
+  rpc StreamJobOutput(JobIdentifier) returns (stream JobOutput) {}
+  // Stops a running job.
+  rpc StopJob(JobIdentifier) returns (StopResponse) {}
+}
+
+// StartRequest contains the job command and rootFS to execute the command in.
+message StartRequest {
+  // The job command line to run.
```

Comment:
Is this executed by a shell?

Reply:
I don't think so. I'm using Go's os/exec package in the part of the library that actually invokes the job command, and the docs for that say _"the os/exec package intentionally does not invoke the system shell and does not expand any glob patterns or handle other expansions, pipelines, or redirections typically done by shells."_

Reply:
Definitely support using `exec` directly and skipping the shell. However, only support command without any args seems quite limiting.

Reply:
I'll add an args array to the gRPC API. In my job runner code, I had a field for command and args in a job attributes struct, but I was unsure whether a combined command and args would be easier to consume than having two distinct fields like os/exec does.

Reply:
Sounds good. There are a number of options here, but without a list, a separate args field, or a shell, I think a single string field would add significant complexity to the implementation. You would need to deal with quotes, whitespace, etc, and document the custom rules.